src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -11,30 +11,46 @@ import {t} from 'app/locale';
 import ExternalIssueStore from 'app/stores/externalIssueStore';
 import getStacktraceBody from 'app/utils/getStacktraceBody';
 import withApi from 'app/utils/withApi';
-
-class SentryAppExternalIssueForm extends React.Component {
-  static propTypes = {
+import {Client} from 'app/api';
+import {Group, PlatformExternalIssue, Event, SentryAppInstallation} from 'app/types';
+import {Field} from 'app/views/settings/components/forms/type';
+
+type Props = {
+  api: Client;
+  group: Group;
+  sentryAppInstallation: SentryAppInstallation;
+  appName: string;
+  config: object;
+  action: 'create' | 'link';
+  event: Event;
+  onSubmitSuccess: (externalIssue: PlatformExternalIssue) => void;
+};
+
+//TODO(TS): Improve typings on Field an input so we can use the type in functions without errors
+
+export class SentryAppExternalIssueForm extends React.Component<Props> {
+  static propTypes: any = {
```

Comment:
What does the `any` do here?

Reply:
@billyvg Good question. This is to ignore the differences between `propTypes` and `Props`. TS doesn't like that we call props like the `sentryAppInstallation` an object in `propTypes` but `Props` has a different type for it. It seems we need it in a lot of components that get wrapped. Just search for `propTypes: any`.